Public/Src/Utilities/UnitTests/Executables/TestProcess/Operation.cs

```diff
@@ -1004,7 +1026,11 @@ private void DoSucceedOnRetry()
         /// Converts command line arg input to process ops
         /// Input format: "OpType?Path?Content?LinkPath?SymLinkFlag"
         /// </summary>
-        public static Operation CreateFromCommandLine(PathTable pathTable, string commandLineArg)
+        /// <remarks>
+        /// This and the execution of Operations within the context of TestProcess.exe should not utilize the PathTable
+        /// since this adds an appreciable amount of JIT overhead to our test execution

```

Comment:
>of JIT overhead [](start = 50, length = 15)

Since we are now mostly on CoreCLR and we can ilgen the assembly. Would this still be the case? Should we just disable this test on full framework?

Reply:
I was thinking on the same lines, but as long as we have most of our customers on full framework i want to keep running the full framework tests. I didn't look into perf implications of this change on NetCore.